### PR TITLE
Ember Node.js LTS Support

### PR DESCRIPTION
**Pull request by nathanhammond**
Current state is ready for review in advance of landing. I'm speaking for the decisions made on the Ember CLI team which primarily lives in the Node.js ecosystem and extrapolating.

/cc @mixonic @ErisDS

/cc Ember CLI @chadhietala @kategengler @kellyselden @rwjblue @stefanpenner @trabus @Turbo87

### CODE REVIEW

File: source/blog/2016-09-01-ember-node-lts-support.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+To make it easier for us to deliver on this promise we have adopted a policy that the `HEAD` of our `master` branches will be compatible with all versions of Node.js currently receiving support of any level. We will not maintain separate branches or classes of support for different Node.js support classifications. Not all of our projects presently align to this pattern but we will, over time, coalesce to this state.
+
+We recognize that we don't have the unilateral privilege of setting this as the default for the entire Node.js ecosystem. It is our hope that with leadership from our community we can encourage more of the Node.js ecosystem to commit to following the schedule laid out by the Node.js LTS Working Group.
+
+Meeting this commitment does not come without cost and tough decisions. However, as maintainers of foundational libraries, we believe that it is in our best interest to validate the legitimacy of the Node.js LTS plans, help position Node.js a reliable and dependable tool for all enterprises, and contribute toward improving the reputation of the ecosystem.
```
Comment (acorncom) [inline source/blog/2016-09-01-ember-node-lts-support.md]: help position Node.js **as** a reliable

File: source/blog/2016-09-01-ember-node-lts-support.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Ember Node.js LTS Support
+author: Nathan Hammond
+tags: Recent Posts, Ember CLI
+---
+
+Ember is committed to being a good steward of our collective codebases as part of the [Node.js](https://nodejs.org/en/) [ecosystem](https://www.npmjs.com/). As such we are fully committed to supporting the [Node.js Long Term Support Schedule](https://github.com/nodejs/LTS#lts-schedule). This means that, per the Node.js Long-term Support Working Group parlance, any "Current", "Active LTS", or "Maintenance" releases may be used for projects within the Ember ecosystem.
```
Comment (locks) [inline source/blog/2016-09-01-ember-node-lts-support.md]: I think "This means that any "Current"…" is enough

File: source/blog/2016-09-01-ember-node-lts-support.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+To make it easier for us to deliver on this promise we have adopted a policy that the `HEAD` of our `master` branches will be compatible with all versions of Node.js currently receiving support of any level. We will not maintain separate branches or classes of support for different Node.js support classifications. Not all of our projects presently align to this pattern but we will, over time, coalesce to this state.
```
Comment (locks) [inline source/blog/2016-09-01-ember-node-lts-support.md]: Can you just say "that our `master` branches"?

File: source/blog/2016-09-01-ember-node-lts-support.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+We recognize that we don't have the unilateral privilege of setting this as the default for the entire Node.js ecosystem. It is our hope that with leadership from our community we can encourage more of the Node.js ecosystem to commit to following the schedule laid out by the Node.js LTS Working Group.
+
+Meeting this commitment does not come without cost and tough decisions. However, as maintainers of foundational libraries, we believe that it is in our best interest to validate the legitimacy of the Node.js LTS plans, help position Node.js as a reliable and dependable tool for all enterprises, and contribute toward improving the reputation of the ecosystem.
+
+### Recommended Version
+
+We will always recommend that you use the most-recent "Active LTS" version of Node.js. At present that is [Node.js v4.5.0](https://nodejs.org/en/download/). This is not to say that you may not use other versions, but by rule the current "Active LTS" will receive the most attention during our testing cycles and be most broadly used within the community.
```
Comment (mixonic) [inline source/blog/2016-09-01-ember-node-lts-support.md]: "We recommend" - who knows what you might do in the future :suspect: 😉

**Comment by nathanhammond** [comment]
Updated and improved. Screenshot also replaced.

**Comment by nathanhammond** [comment]
Addressed feedback from @mixonic provided via Slack, squashed, changed publish date, and added example for Ember CLI support model. Last call!

**Comment by mixonic** [comment]
👍 :shipit:

**Comment by nathanhammond** [comment]
So let's ship it!

**Comment by chadhietala** [comment]
LGTM

**Comment by stefanpenner** [comment]
👍

**Comment by nathanhammond** [comment]
Rebased with September 7 as the post date. Ship?

**Comment by ErisDS** [comment]
Sorry I'm late!

I absolutely love this 😍 . Thank you so much @nathanhammond for taking my ideas on board and so seriously. I believe this is model citizenship ❤️  Huzzah 🎉

**Pull request merged**